scripts/RotaryEncoder.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python3
+# rotary volume knob
+# these files belong all together:
+# RPi-Jukebox-RFID/scripts/rotary-encoder.py
```

Comment:
Having files RotaryEncoder.py and rotary-encoder.py may be confusing or do you intend to merge these files?